packages/ui-extensions/src/surfaces/point-of-sale/components/POSBlock/POSBlockRow.ts

```diff
@@ -0,0 +1,17 @@
+import {createRemoteComponent} from '@remote-ui/core';
+
+/**
+ * Renders a `POSBlockRow` in a `POSBlock`.
+ */
+export interface POSBlockRowProps {
+  /**
+   * Whether to remove the default padding from the `POSBlockRow`.
+   * @defaultValue false
+   */
+  removeInsets?: boolean;
```

Comment:
Just remembered that we're also applying vertical padding by default as well, and this prop will only remove the horizontal ones (vertical ones should not be modifiable imo). Should we name this `removeHorizontalInsets` to be more clear?